scylla-cql/src/value.rs

```diff
@@ -779,3 +791,461 @@ pub struct CqlDuration {
     pub days: i32,
     pub nanoseconds: i64,
 }
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum CqlValue {
+    Ascii(String),
+    Boolean(bool),
+    Blob(Vec<u8>),
+    Counter(Counter),
+    Decimal(CqlDecimal),
```

Comment:
:memo: Variants are missing docstrings, too, but these should be easily searchable in the ScyllaDB documentation.